### PR TITLE
Fixed auth health check when auth is disabled and Swagger api

### PR DESCRIPTION
**Pull request by holashchand**

### CODE REVIEW

File: java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SecurityConfig.java [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
 @Configuration
 @EnableWebSecurity
-@ConditionalOnProperty(name = "authentication.enabled",havingValue = "true",matchIfMissing = false)
```
Comment (srprasanna) [inline java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SecurityConfig.java:23]: Why is this removed.

File: java/registry/src/main/resources/public/swagger-ui.html [hunk 2]
```diff
@@ -37,7 +37,7 @@
     window.onload = function() {
         // Begin Swagger UI call region
         const ui = SwaggerUIBundle({
-            url: "/api/docs/swagger.json",
+            url: window.location.pathname.split("/").slice(0, -1).join("/") + "/api/docs/swagger.json",
```
Comment (srprasanna) [inline java/registry/src/main/resources/public/swagger-ui.html:40]: Can you make the change to support for relative URL instead of absolute.

**Review by srprasanna — COMMENTED** [review]
@holashchand  test cases are missing for the change. Could you add them as well.

File: java/registry/src/main/resources/public/swagger-ui.html (hunk 2 above)
Comment (holashchand) [reply java/registry/src/main/resources/public/swagger-ui.html:40]: The change is to make it a relative url only, to get the context url `window.location.pathname.split("/").slice(0, -1).join("/")` is added.
i.e. if swagger-ui endpoint is `https://api.abc.com/registry/swagger-ui.html `then it adds the` /registry` before the api json endpoint

File: java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SecurityConfig.java (hunk 1 above)
Comment (holashchand) [reply java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SecurityConfig.java:23]: Got it, rolled this back

**Review by srprasanna — APPROVED** [review]
